Vostok.Applications.AspNetCore/Configuration/UnhandledErrorsSettings.cs

```diff
@@ -0,0 +1,17 @@
+﻿using JetBrains.Annotations;
+using Vostok.Applications.AspNetCore.Middlewares;
+
+namespace Vostok.Applications.AspNetCore.Configuration
+{
+    /// <summary>
+    /// Represents configuration of <see cref="UnhandledErrorMiddleware"/>.
+    /// </summary>
+    [PublicAPI]
+    public class UnhandledErrorsSettings
+    {
+        /// <summary>
+        /// Error response code to be used when an unhandled exception is observed.
+        /// </summary>
+        public int ErrorResponseCode { get; set; } = 500;
```

Comment:
I think it would be cool to control the error code depending on the exception (500 or 503) 

```suggestion
        public Func<Exception, int> ExceptionResponseCode { get; set; } = _ => 500;
```

Reply:
IMO this middleware shouldn't have such level of customization as it may lead to exceptions (e.g. in functor which you suggested). Response code/body/etc. should be mapped by client's own code.

I think the purpose of this middleware is to only intercept exception before logging it at kestrel level. As logging in kestrel won't include traceId (which is no longer available in flowing context at that execution site)